[15.0][ADD] purchase_order_line_menu: New views and stored fields (#1928)
OCA/purchase-workflow: purchase_order_line_menu/readme/DESCRIPTION.rst

> Adds a menu item and a couple of views to navigate through Purchase Order lines.

Reviewer: ```suggestion
Adds a menu item and some views to navigate through Purchase Order lines.
```
